### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by LifeGold** [Virtual-Universe/Virtual-Universe#35]
Fix Sorry

**Comment by emperorstarfinder** [comment]
I am assuming this is related to your Issue #34 if so please try to remember to comment in that issue if you did fix it to avoid confusion.  Thanks

**Comment by LifeGold** [comment]
To compile the server. Launched, in this error he wrote you for the first time launched a generic on this computer. if you do not set up the configure files.
logs
https://cloud.mail.ru/public/6TzC/ZBk7RYTgC

**Comment by emperorstarfinder** [comment]
and he moved this to a different server? if so then you might need to do the run configurator command and update the url to the grid server on Universe.sh if your running in grid mode.

**Comment by LifeGold** [comment]
I downloaded and put on a new server without the old files

**Comment by emperorstarfinder** [comment]
Okay do the command run configurator if your trying to hook it up to a grid server answer the questions and then once the configurator is completed restart the region server that should solve that as it looks like it cant access your grid service.  

[WebUtils] request failed: System.Net.WebException: Error: ConnectFailure (Connection refused) 

That message usually means the server has actively refused the connection.  It may also be due to ports being used currently.  In most cases if your on a linux server then rebooting the machine will solve that if its Ubuntu.  On windows you need to open the ports in the windows firewall and in the modem/ router.  server machines provided by a hosting provider may also have some ports blocked but usually not the ports youll need.

My guess is if your still on Ubuntu restarting the machine should resolve it.  If not do the run configurator command on the region server and that should update your urls needed.

**Comment by LifeGold** [comment]
Error strange issues, once turned off the server issue has now disabled the server it was not.

**Comment by emperorstarfinder** [comment]
I assume your still running on Ubuntu

**Comment by LifeGold** [comment]
Yes

**Comment by emperorstarfinder** [comment]
well the connection refused error usually means a port is not open but by default Ubuntu does not have an enabled firewall so that would have suggested another service running on that port

**Comment by LifeGold** [comment]
Most interesting what I have is pure Ubuntu, and installed programs only mono and xampp

**Comment by emperorstarfinder** [comment]
hmmmmm i seriously doubt xampp would cause that as it doesnt use ports in the 8000-8999 range except it does have the ability to use port 8080 which is not one we use so i dont think that would be your offending application but stranger things have happened

**Comment by LifeGold** [comment]
But in Ubuntu how to open ports?(

**Comment by emperorstarfinder** [comment]
By default Ubuntu does not have ports blocked so you shouldnt need to do anything within ubuntu itself  so it could very well be something got blocked by your host provider which is a port you need open for your ip addresses of your servers.  Also you should use your FQDN (Fully Qualified Domain Name) instead of your ip in Universe.Server.ini and Universe.ini that way if the ip changes your regions still will work properly because it uses the FQDN 

As I said ill do some digging after dinner and see if there is something going on that is being caused by Virtual Universe but I am pretty confident its relating to something else.

**Comment by LifeGold** [comment]
Checked the port 8080, 8000, 9001, 9002, they closed all(

**Comment by emperorstarfinder** [comment]
then thats why.  you need ports the ranges of 8000-8999 and 9000-999 TCP/UDP open (in the range of 8000-8999) you technically only need ports 8000-8013 for Virtual Universe unblocked for both incoming and outgoing but i do the entire range so i dont have to worry about it down the road.

**Comment by emperorstarfinder** [comment]
Closing as this appears to be an issue caused by the user's hosting provider.  If this persists after the hosting provider corrects the error in their network the user can open a new issue with the exact error using the built in issue reporting template.

Emperor Starfinder
CEO/ Founder
Core Developer
Work Group Leader
Grid Architecture Development Work Group
Second Galaxy Development Team
